pkg/products/threescale/reconciler.go

```diff
@@ -850,6 +850,9 @@ func (r *Reconciler) reconcileOpenshiftUsers(ctx context.Context, installation *
 
 	added, deleted := r.getUserDiff(kcu, tsUsers.Users)
 	for _, kcUser := range added {
+		if kcUser.Email == "" {
+			kcUser.Email = fmt.Sprintf("%v@example.com", kcUser.UserName)
```

Comment:
Did this end up being required for a product in RHPDS?

Reply:
Ah, I added this once I saw that it was possible to create a user without an email address, not because I saw a product required one. How would I check whether each product needs an email address? Try to log into each product?